source/codegen/metadata/nifake_non_ivi/functions.py

```diff
@@ -146,4 +146,33 @@
         ],
         'returns': 'int32'
     },
+    'RegisterCallback': {
```

Comment:
Note: I decided to only test this with a synchronous callback instead of  streaming callback, The streaming implementation would require faking a grpc ServerWriter. Since that is not part of the final implementation, that would be wasted work.